dev.rollup.config.js

```diff
@@ -0,0 +1,66 @@
+/* eslint-env node */
+'use strict';
+
+import pkg from './package.json';
+import os from 'os';
+import fs from 'fs-extra';
+import path from 'path';
+import glob from 'glob';
+import serve from 'rollup-plugin-serve';
+import livereload from 'rollup-plugin-livereload';
+import { umdConfig } from './rollup.config';
+
+const BROWSER_FILENAME = `${pkg.name}.js`;
+const tmpDestPath = fs.mkdtempSync(path.join(os.tmpdir(), `${pkg.name}-dev-server-`));
+
+const httpsConfig = {
+    key: fs.readFileSync('sslcert-dev/key.pem'),
```

Comment:
Should we wrap this with try/catch and either warn and fall back to http or fall if these files don't exist?  Not sure if any of these plugins strictly require https.  That would make the choice easy.

Reply:
Yeah, that's a good call. I don't believe that either of the plugins requires https so warning and falling back to HTTP would be a good approach here.